### ISSUE TITLE
(WIP) Add 'Support Opentoonz' in Help Menu

### ISSUE THREAD

**Issue by RodneyBaker** [opentoonz/opentoonz#2939]
A popular topic of late has been that of supporting Opentoonz (especially financially) as well as helping users know how to contribute by reporting bugs, feature requests etc.

Please reference  #2566 for a recent discussion related to this and a possible way forward accounting for the v1.4 release.

In Summary:

- Create two landing pages/sections in Opentoonz documentation for two additional links in the Help Menu dropdown.  These would include:

- Support Opentoonz
- Report Bug   (or Report a Bug etc as preferred)

This will help set the stage for success moving into 2020, v1.5 and beyond.

**Comment by shun-iwasawa** [comment]
I asked Dwango for their thoughts about collecting donations.
According to Keiichi Suzuki, CTO of Dwango;

> We have no plans to collect donations at this point in time. At present, we estimate that the disadvantages such as burden of management are more than the advantages of collecting money.


I have the same opinion about it.
If some financial reward will be effective to motivate or attract developers, I think it'd better to focus on BountySource, which is much clearer in management, from whom to who the money will go for what purpose.

Based on the notion above, I'm sorry to say but am hesitant to include the menu labeled `Support OpenToonz` in the software at this point as users may misunderstand there is (or it is planned to have) a way to donate.

**Comment by RodneyBaker** [comment]
Thanks @shun-iwasawa 
Your feedback on this is much appreciated as is that of Keiichi Suzuki.

As I was the one who suggested using the term 'support' rather than 'donation' I'll add a little more context to that suggestion:

- If a landing page places emphasis on support in the form of programming, development, documentation and testing this echoes other similar goals stated in other official Opentoonz documentation.  For instance, as seen on the primary Opentoonz website and via github.  

- The quote you share from Keiichi Suzuki would be very welcomed on a support page.  Users might not agree with that direction but at least they would know.  This would let those that wish to support Opentoonz financially that donations are not being accepted and there are no current plans to do so.

- A statement such as, "Those that wish to directly support developers may consider contributing via BountySource, Patreon, Github Sponsors or the developers' preferred resource." would certainly be welcome to suggest how those that wish to contribute financially might move forward.


I'll add that an original goal of Opentoonz/Dwango is stated on the Opentoonz website as:

> We aim to develop a new platform for connecting academic research with cutting edge animation production.

Assuming this is still true, it would be good to highlight this goal on the support page also to continue to encourage those in academia to support development of Opentoonz.  (I assume government grants and studio/commercial production support, etc. would fall into this category also)

In short, although officially Dwango doesn't desire direct financial donations from users, support is still desired in other forms from individual contributors.  This is an opportunity to emphasize the ideal form of that support while communicating to those interested that they have been heard.

I'm not sure I have a particular preference one way or the other regarding adding a Support link in Opentoonz... other than to suggest the OT community doesn't have the infrastructure for financial donations... but am responding to increasing interest from individuals in the Opentoonz community.  It seems likely to me that if the official release doesn't have a Support link we may see the link appear in other Opentoonz forks.  Having the link go to an official Support page would be preferred.

**Comment by shun-iwasawa** [comment]
I think I understand your suggestion that the link will be still helpful for users. 
Then can you please try creating the suggested page in the official website `opentoonz.github.io` and post a PR for review, instead of creating it in the documentation page ?
As you know, currently the documentation page is quite open for anybody in order to speed-up the improvement. I think the `Support OpenToonz` page is rather needed to be carefully reviewed to minimize the future risk.

**Comment by RodneyBaker** [comment]
@gab3d An official Support page via opentoonz.github.io seems ideal for initiating an official support link via Opentoonz Help menu.  What do you say to assisting with placement there?

**Comment by RodneyBaker** [comment]
I've started on an initial Support page with the idea to break it down into two main categories:

- Getting Support
- Contributing Support

The focus of 'Getting Support' will be toward online documentation and the Opentoonz Forum with highlight on the present internal link inside Opentoonz to the readthedocs site.

The 'Contribute Support' section to focus on github (development and testing processes) and direct support of developers (BountySource).

![support_page](https://user-images.githubusercontent.com/2576432/71469384-8b0dd400-278e-11ea-8769-ab50f437a631.png)

**Comment by gab3d** [comment]
@shun-iwasawa @RodneyBaker 
While a little sad about the outcome, I can say I certainly understand that there would be an administrative work to be done for handling a donations system. By intuition I'd say that the benefits of the donated resources would outpace the administrative burden, but of course I can't prove it, so... 

Regarding the **Support OpenToonz** page in the official site, I'm all for it. 
It'll give a clear path for users wanting to support OT in any form (even if, currently, that support may not take the form of money donation), amd that kind of things help make the community stronger.
While I'm by no means an expert in page creation I could try to help with that, if it's needed. 👍 

I don't envision that page having content related to documentation, though... 
I mean, I think people will arrive to the **Support OpenToonz** page mainly from the program itself, when they are curious about how to provide "Support for OpenToonz" in any form. And while the terms repeat, that is not the same as when you are looking for "OpenToonz Support", if I'm explaining right... 🤔
 
That said, I do wish there could be a "Documentation" or "Learn OpenToonz" page inside the official site, where leads could be given on how to find good learning material about OpenToonz (the Manual itself, sample scenes, perhaps even an official YouTube channel could be created in the future to give tips on new features and a link could be placed there too, who knows... if someone makes a book on OT help promoting it there, etc.)

**Comment by RodneyBaker** [comment]
Attached please find an initial draft of a Support page.
Links to resources haven't been added.
This attempt is mostly just to get a simple page drafted.

Download and unzip
Enter e directory (english)
Open index.html (or Support.html)

[opentoonz.github.io- Support Page draft.zip](https://github.com/opentoonz/opentoonz/files/4003395/opentoonz.github.io-.Support.Page.draft.zip)

I removed other various files in order to limit file size of upload.

Key points:
- Two basic tracks to follow for Support:  Getting Support and Contributing Support
- Where to find more information (primarily OT Forum and Github)
- Recommendation that the best way to contribute to Opentoonz community is through usage of Opentoonz.

**Comment by RodneyBaker** [comment]
@gab3d said:

>  I do wish there could be a "Documentation" or "Learn OpenToonz" page inside the official site

That gives us a good path forward for 2020 but we need to lock v1.4 down first.  :)

**Comment by manongjohn** [comment]
FYI, I'm with @shun-iwasawa on being hesitant to put any `Support OpenToonz` button in OT's Help Menu.

I'd only put one if there is an actual official donation page that accepts currency, someone to manage the donations and a clear and fair outline of how those donations are to be allocated to developers.

**Comment by gab3d** [comment]
I can certainly understand the concerns, nevertheless there are also other ways to support the project: programming, documenting, translating, etc. and no money involved in that.

**Comment by RodneyBaker** [comment]
Based on @manongjohn 's PR #2966 I believe this request can be closed.
What say you @gab3d ?

**Comment by gab3d** [comment]
I believe so... thanks.
